ubireader/ubi_io.py

```diff
@@ -192,11 +200,11 @@ def __init__(self, ubi, block_list):
             self._blocks = sort.by_leb(block_list)
             self._seek = 0
             self._last_leb = -1
-            self._last_buf = ''
+            self._last_buf = b''
```

Comment:
this is the only actual code change in this PR I saw.
I think you are right with this change, but please revert it to have no non-typing changes.

Reply:
You're right, I forgot to revert this.